ReSwiftThunk.podspec

```diff
@@ -20,7 +20,19 @@ Pod::Spec.new do |spec|
   spec.source = {
     :git => "https://github.com/ReSwift/ReSwift-Thunk.git",
     :tag => spec.version.to_s }
-  spec.source_files = "ReSwift-Thunk"
+
+  spec.subspec "Core" do |sp|
+    sp.source_files = "ReSwift-Thunk"
+  end
+
+  spec.subspec "ExpectThunk" do |sp|
+    sp.dependency "ReSwiftThunk/Core"
```

Comment:
If you remove this dependency, perhaps the inherited podfile example will work? I think it might be double-importing `Core` when `inherit` is included in the test target pod specifications.

Reply:
Good thought, however, I wasn't able to get it to build this way. I'm attempting with CocoaPods 1.6.1. It may be an open issue with CocoaPods [7195](https://github.com/CocoaPods/CocoaPods/issues/7195) having child targets with subspecs. I'll follow up there, it does seem splitting the targets is the workaround in the Podfile at the moment.